Makefile

```diff
@@ -36,7 +38,14 @@ include buildenv.mk
 all: dcap_source sdk psw
 
 dcap_source:
+ifeq ($(shell git rev-parse --is-inside-work-tree), true)
 	git submodule update --init --recursive
+else
+	curl --output dcap_source.tar.gz -L --tlsv1 ${DCAP_DOWNLOAD_BASE}/DCAP_${DCAP_VER}.tar.gz
```

Comment:
DO NOT download during Make.

Please add this to the script that downloads dependencies prior to build. instead.

Reply:
Sorry for the late response, I've missed to answer to your comment. Then, should this change be reverted?